### ISSUE TITLE
Allow claim search parameter to ignore filtered claims / channels

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-sdk#2740]
If this parameter is passed, ignore any rules set by the FILTERING_CHANNEL_IDS wallet server parameter.

**Comment by lyoshenka** [comment]
i don't think we can do this. filtering is done by the hub operator. it doesn't make sense to allow users to just bypass filtering when they want to.

**Comment by tzarebczan** [comment]
This is not the blocking feature, but rather the filtering one, i.e. unmarked porn, or anything in the future attribution system.
